templates/windows/chocolateyInstall.ps1.erb

```diff
@@ -0,0 +1,5 @@
+<%- dirnames = get_directories.map {|d| d.path } -%>
+<%- dirnames.each do |dir| -%>
+  <%- windows_path = platform.convert_to_windows_path(dir) -%>
+Copy-Item <%= "#{windows_path.sub("C:", "C:/ProgramData/chocolatey/lib/#{project.name}")}" %> <%= windows_path %> -recurse
```

Comment:
@MikaelSmith are you referring to not wanting to unzip multiple times during install? 

@ferventcoder what if we don't have a zip file? Should I stick with `Copy-Item` and iterate over our list of directories?